xmipp

```diff
@@ -636,6 +653,23 @@ def compileModule(Nproc,module,printFinish=False):
         stampVersion()
     return ok
 
+def compileDependencies(Nproc):
+    print("Building Dependencies -------------------------------------")
+    if not checkProgram("nvcc",False):
+        print(red('Dependencies not installed as they need NVCC, which has not been found in the PATH.\nDepending on other settings, build might crash eventually.'))
+        return True
+
+    advisorDir = "src/cuFFTAdvisor/" 
+    currDir = os.getcwd()
+    libDir = "src/xmipp/lib/"
+    createDir(libDir)
+    os.chdir(advisorDir)
+    ok = runJob("make all")
+    os.chdir(currDir)
+    ok = ok and runJob("cp " + advisorDir + "build/libcuFFTAdvisor.so" + " " + libDir)
```

Comment:
in the build root folder we don't have compiled code. Additionally, the build folder shouldn't be hard coded, because the user can choose his/her own installation destination.

Reply:
This is build of the external repo.
The idea is that xmipp downloads the dependecies, build them (in the way the dependencies want), and than cherry-pick whatever is necessary, or link against them.

Reply:
Sorry, I misunderstood the command... I saw `cp wherever build/blabla.so` instead of `cp whatever/build/blabla.so libDir` my apologies. For sure, the dependency can build the code as it wants XD.